lib/dslash_quda.cu

```diff
@@ -88,31 +88,31 @@ namespace quda {
 
 #if CUDA_VERSION >= 8000
     cuuint32_t *commsEnd_h;
-    CUdeviceptr commsEnd_d[Nstream];
+    QUdeviceptr commsEnd_d[Nstream];
```

Comment:
This is a note to self.  This is a remnant from some experiments with async policies that never really worked.  I can just delete them.